refarch-gateway/src/test/resources/application-test.yml

```
@@ -39,3 +30,8 @@ spring:
           keycloak:
             client-id: refarch-gateway
             client-secret:
+
+wiremock:
+  server:
+    port: 0
+    https-port: -1
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)